content/zh/blog/exceptions.md

```diff
@@ -0,0 +1,181 @@
+---
+标题: Falco 规则现在支持例外
```

Comment:
Don't translate the metadata keys please

Reply:
OK, I'll revise it.